boards/arm/nucleo_h743zi/nucleo_h743zi.dts

```diff
@@ -90,3 +90,11 @@
 &rng {
 	status = "okay";
 };
+
+&spi1 {
+	status = "okay";
+};
```

Comment:
Thanks for contribution. 
Please use the new SPI pinctrl has to be defined from device tree:
```suggestion

&spi1 {
	status = "okay";
	pinctrl-0 = <&spi1_nss_pb14 &spi1_sck_pa5 
	             &spi1_miso_pa6 &spi1_mosi_pb5>;
};
```

Reply:
pb14 is not a nss pin in stm32h743zitx-pinctrl.dtsi, so I moved it to pa4

Reply:
Alright then, just gave you the basic syntax ;-)

Reply:
As I said in another comment, I don't think you can bring Nucleo_H743ZI SPI support yet since the H7 series is not supported by the Zephyr STM32_LL_SPI driver. But maybe SDMMC could be ok, I'll try to check that.